package.json

```diff
@@ -25,6 +25,9 @@
     "browser",
     "sdk"
   ],
+  "dependencies": {
+    "@fullstory/snippet": "1.3.0"
```

Comment:
also nit: the https://www.npmjs.com/package/@fullstory/snippet doesn't have any README yet. Now that we depends on it, may be a good idea to have add the README for clarity?

Reply:
Hi @sabrina-li!  Thanks for the review.

The idea is for the two of these packages to be tightly coupled. Our bionic update strategy would be to use dependabot to automatically set up PR's for updating. (I might need to do a small amount of research on the requirements for setting this up as I'm not sure if dependabot needs there to be the `^` to know whether it can open a PR 🤔 )

RE: open source - the `@fullstory/snippet` package is the code we deemed safe for the public. Originally there was a discussion about some sort of hook that published the snippet code into this SDK, but there were enough additional use cases for a snippet-only package that we didn't want to cloud the intent of this package. Specifically, `@fullstory/snippet` provides a "stringified" version of the snippet which is useful for some build-system integrations like injecting the snippet into an html template in a webpack config. 

That said, because `@fullstory/browser` is open source, some might be curious about the new dependency. I will put together some documentation for that and publish a new version before merging in this PR. 

I'm going to put this PR back in draft while I take care of that and I'll ping you when it's done!

And to you first point, after we bump to snippet v2, the hope is that `@fullstory/snippet` won't need to be updated for quite a while. 🤞

Reply:
Thank you for the context! 
I was floating ideas around using git submodules to push to OSS from within mn (it's only an idea at this point as we don't have any real use cases for it yet). So was just curious if we plan on making the `@fullstory/snippet` open source.